src/molecules/formfields/CreditCardField/index.js

```diff
@@ -48,13 +52,11 @@ function CreditCardField(props) {
           <div className={styles['col']}>{ securityCode }</div>
         </div>
       </div>
-      { meta && meta.touched && meta.error &&
-        <div className={styles['error']}>{ meta.error }</div>
-      }
 
-      {meta && meta.touched && meta.warning &&
-        <div className={styles.error}>{ meta.warning }</div>
-      }
+      <ErrorMessage
+        condition={!!message}
```

Comment:
the condition is necessary for `TransitionGroup` to fire its lifecycle methods. it's all dependent on a component being added/removed from the DOM, and the condition is what controls that